templates/database.yml.j2

```diff
@@ -0,0 +1,9 @@
+# {{ ansible_managed }}
+production:
+  adapter: postgresql
+  encoding: unicode
+  database: {{ gitlab_db_name }}
+  pool: {{ gitlab_db_pool }}
+  username: {{ gitlab_db_user }}
+  host: 'localhost'
+  socket: {{ postgres_unix_socket_directories }}
```

Comment:
What if I want to connect to a remote postgres?

Reply:
fixed